content/technical-guide/integrations/engine.md

```diff
@@ -188,6 +188,11 @@ For general information about the configuration via Spring XML see the
     <th>Cawemo Engine Plugin</th>
     <th>Camunda Platform Engine</th>
   </tr>
+  <tr>
+    <td>1.8</td>
+    <td>1.1.x or newer</td>
```

Comment:
@jfriedenstab didn't we release some changes in the engine plugin recently? Is this still true? All changes are backwards compatible?